sros2/sros2/api/_key.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019-2020 Canonical Ltd
```

Comment:
Nit: I think @mikaelarguedas commented on this same copyright notice from last PR.

Reply:
This one should be legit; I wrote some of these functions and modified others.